README.md

```diff
@@ -1,22 +1,5 @@
-# Network Upgrade Process - Eth1.0 Specification
+# Eth1.0 Specifications
 
 ## Description
-This provides a general outline process to propose, discuss, and track the progress of EIPs considered for network upgrades to Ethereum Mainnet.
 
-## Process
-
-### Preconditions for Consideration
- - Type of EIP -  Core
- - Status of EIP - Any status between Draft to Final, see [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips) for more details.
-
-### Proposing an EIP
-
-1. If you're an author, and still vetting the idea, please follow the guidelines mentioned in [EIP-1](https://eips.ethereum.org/EIPS/eip-1) to formalize your idea into an EIP.
-1. Once an EIP is created (or a pull request is open against the [ethereum/eips](https://github.com/ethereum/EIPs/pulls) repository), [open a new issue in this repository](https://github.com/ethereum/eth1.0-specs/issues/new) with a title of the form "EIP-XXXX to be Considered for Inclusion".
-1. The EIP will then be added to the "CFI Applied" column in [this project tracker](https://github.com/ethereum/eth1.0-specs/projects/1).
-1. For your EIP to move forward in the network upgrade process, it should be discussed on the AllCoreDevs call. To propose this, open an issue against the [`ethereum/pm` repository](https://github.com/ethereum/pm/issues/new). It is not required to be part of the project tracker to be discussed in an ACD meeting.
-
-### Socializing an EIP
-- Open a discussion thread preferably at the [Fellowship of Ethereum Magicians](https://ethereum-magicians.org/) to gather feedback from the community. 
-- Join the Ethereum Cat Herders' ['Peep an EIP'](https://github.com/ethereum-cat-herders/PM/projects/2#column-10491086) video series to discuss the background, need, and application of the EIP followed by Q&A.
-- Invite the author to [Ethereum All core dev meetings](https://github.com/ethereum/pm#all-core-devs-meetings) for EIP to be introduced to the client teams.
+This repository contains various specification related to the Ethereum 1.0 chain, specifically the specifications for [network upgrades](/network-upgrades) and (soon) the [JSON RPC API](/json-rpc). 
```

Comment:
```suggestion
This repository contains the Ethereum Execution Layer (Eth1.0) network upgrades and (soon) the JSON RPC API specifications.
```

Reply:
Thanks for catching the typo! Re: eth1 vs. Execution Environment, I think I want to keep the eth1 terminology here for now given the repo is called eth1-specs. When we merge this repo and the eth2 specs repo, I think we should change the terminology.

Reply:
Cool! Looks like I missed on the repo link that is there in your earlier texts :)